dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/AclEventExporterTest.java

```diff
@@ -369,11 +373,37 @@ void shouldReturnSelectedOrgUnitEventsWhenNoOuModeSpecifiedAndUserHasAccessToOrg
                     + " instead"));
   }
 
+  @Test
+  void shouldReturnAccessibleOrgUnitEventsWhenNoOuModeNorOrgUnitSpecifiedAndUserHasAccessToOrgUnit()
+      throws ForbiddenException, BadRequestException {
+    injectSecurityContext(userService.getUser("FIgVWzUCkpw"));
+    EventOperationParams params =
+        EventOperationParams.builder().programUid("pcxIanBWlSY").orgUnitMode(ACCESSIBLE).build();
+
+    List<Event> events = eventService.getEvents(params).getEvents();
+
+    assertFalse(
+        events.isEmpty(),
+        "Expected to find accessible org unit events when ou mode nor org unit specified");
+    events.forEach(
+        e ->
+            assertEquals(
+                "uoNW0E3xXUy",
+                e.getOrganisationUnit().getUid(),
+                "Expected to find accessible org unit uoNW0E3xXUy, but found "
+                    + e.getOrganisationUnit().getUid()
+                    + " instead"));
+  }
+
   @Test
   void shouldFailWhenNoOuModeSpecifiedAndUserHasNoAccessToOrgUnit() {
     injectSecurityContext(userService.getUser("FIgVWzUCkpw"));
     EventOperationParams params =
-        EventOperationParams.builder().programUid("pcxIanBWlSY").orgUnitUid("DiszpKrYNg8").build();
+        EventOperationParams.builder()
+            .programUid("pcxIanBWlSY")
+            .orgUnitUid("DiszpKrYNg8")
+            .orgUnitMode(SELECTED)
```

Comment:
The test name is 

`shouldFailWhenNoOuModeSpecifiedAndUserHasNoAccessToOrgUnit`

should probably be something like

`shouldFailWhenUserHasNoAccessToSelectedOrgUnit`

Reply:
Yeah. Anyway, with the latest changes this test is not useful anymore.